cms_static.py

```diff
@@ -11,7 +11,7 @@
 CMSBUILD_GH_USER = "cmsbuild"
 CMSBOT_IGNORE_MSG = "<cmsbot>\s*</cmsbot>"
 CMSBOT_NO_NOTIFY_MSG = "<notify>\s*</notify>"
-CMSBOT_TECHNICAL_MSG = "cms-bot internal usage"
+CMSBOT_TECHNICAL_MSG = "cms-bot internal usage(\d*)"
```

Comment:
@iarspider , I think we do not need this change. Technical comments will be in order any way. So no need to add `(index#)`